ENIGMAsystem/SHELL/Graphics_Systems/Direct3D9/DX9ModelStruct.h

```diff
@@ -16,23 +16,16 @@
 **/
 
 #include "Bridges/General/DX9Context.h"
-#include "../General/GSd3d.h"
-#include "../General/GSprimitives.h"
+#include "Graphics_Systems/General/GSd3d.h"
+#include "Graphics_Systems/General/GScolor_macros.h"
+#include "Graphics_Systems/General/GSprimitives.h"
 #include "Universal_System/var4.h"
 #include "Universal_System/roomsystem.h"
 #include <math.h>
 #include <stdlib.h>
 
-
 using namespace std;
```

Comment:
burn it with fire

Reply:
I will, in the next/future pr, but I may do bridges like you want first. Regardless, this has no bearing on this pr.